tests/kafkatest/tests/connect/connect_distributed_test.py

```diff
@@ -418,7 +418,7 @@ def test_bounce(self, clean):
         if not success:
             self.mark_for_collect(self.cc)
             # Also collect the data in the topic to aid in debugging
-            consumer_validator = ConsoleConsumer(self.test_context, 1, self.kafka, self.source.topic, consumer_timeout_ms=1000, print_key=True)
+            consumer_validator = ConsoleConsumer(self.test_context, 1, self.kafka, self.source.topic, new_consumer=False, consumer_timeout_ms=1000, print_key=True)
```

Comment:
Hmm, I'm not sure. To be honest, I did not pay close attention to the logic behind the tests. I just made sure if it used the old consumer before, it should still use the old consumer. It's not obvious to me from the test itself.